CONTRIBUTING.md

```diff
@@ -18,29 +18,30 @@
 [license-image]: https://img.shields.io/badge/license-CC_BY_4.0-blue.svg?label=License
 
 Contributions to the tldr-pages project are [most welcome](GOVERNANCE.md)!
-All `tldr` pages are stored in Markdown right here on GitHub.
-Just open an issue or send a pull request and we'll incorporate it as soon as possible.
+
+All `tldr` pages are stored in Markdown right here on GitHub. Just open an issue or send a pull request, we'll incorporate it as soon as possible.
```

Comment:
Perhaps this will read better? 
```suggestion
All `tldr` pages are stored in Markdown right here on GitHub. Just open an issue or send a pull request, and we'll incorporate it as soon as possible.
```